doc/src/tutorial/compile_windows_mingw.dox

```diff
@@ -7,6 +7,7 @@
 a draft of new MinGW information on our
 Wiki: <a href="https://github.com/PortAudio/portaudio/wiki/Notes_about_building_PortAudio_with_MinGW">
 PortAudio Wiki: Notes about building PortAudio with MinGW</a></strong>
+If you are new to MinGW, this is the wrong page, and the correct page is almost certainly \ref compile_windows_mingw-w64. (MinGW is a largely defunct and unused project; MinGW-w64 is its current iteration.)
```

Comment:
"this is the wrong page" is too strong. We need to be neutral, avoid making value judgements, and and let the user choose. Try:

"If you are new to MinGW, consider MinGW-64: \ref compile_windows_mingw-w64. It is a newer project.

Reply:
I made the requested change, but I disagree completely. This is doing a disservice to users. MinGW has been defunct for a long, long time, and its entire purpose nowadays is to trick the unwary. I do not think MinGW's continued existence is even morally right, and it reflects negatively upon its maintainer Keith. Mentioning it in PA's docs as an option is similar to teaching Flat Earth theories in school.

For example, here is someone celebrating the disappearance of the mingw.org domain: ["It's probably for the best that mingw.org is down - that version is essentially abadoned and it's frustrating that they didn't admit that on their homepage many years ago."](https://old.reddit.com/r/cpp/comments/kuyqz1/mingworg_is_down/giv0ges/)